src/main/webapp/scripts/navbar.js

```diff
@@ -37,6 +37,10 @@ function getActiveTab() {
 
 function setActiveTab(fileName) {
   const activeTabElement = document.getElementById(`${fileName}_tab`);
+  // If its View Card then return
```

Comment:
Can this comment be more descriptive? I did not understand what this is trying to say